design/FY2023/NFP-evapCoolerRHcontrol.md

```diff
@@ -0,0 +1,96 @@
+
+The introduction of excessive moisture is one of the potential issues of direct evaporative coolers. A humidity control could become useful in preventing the evaporative cooler from raising indoor humidity to an uncomfortable level. Currently, the direct evaporative cooler can be controlled with the sensor node  temperature using AvailabilityManagers (*AvailabilityManager:LowTemperatureTurnOff* or *AvailabilityManager:HighTemperatureTurnOn*). This feature proposes to add a relative humidity (RH) control to shut down the evaporative cooler when the indoor RH is too high.
+
+## Approach
+
+To enable a RH control, two *AvailabilityManager:\** objects will be added *AvailabilityManager:HighRHTurnOff* and *AvailabilityManager:HighTemperatureLowRHTurnOn*.
+
+## Testing/Validation/Data Source(s)
+
+This feature will be tested and demonstrated with a test file derived from 1ZoneEvapCooler.idf. Manual check of the time-step EnergyPlus simulation results will be conducted to confirm the added feature is working correctly. 
+
+## IDD Object changes
+
+Two *AvailabilityManager:\** objects, *AvailabilityManager:HighRHTurnOff* and *AvailabilityManager:HighTemperatureLowRHTurnOn* will be added to enable RH control of the direct evaporative cooling.
+
+    AvailabilityManager:HighRHTurnOff,
+          \memo Overrides fan/pump schedules depending on relative humidity at sensor node.
+      A1 , \field Name
+          \required-field
+          \type alpha
+          \reference SystemAvailabilityManagers
+      A2 , \field Sensor Node Name
+          \note The air node where the relative humidity is monitored
+          \required-field
+          \type node
+      N1 ; \field Relative Humidity
+          \note The relative humidity at or exceeding which the system is turned off
+          \required-field
+          \type real
+          \units dimensionless
+
+    AvailabilityManager:HighTemperatureLowRHTurnOn
+          \memo Overrides fan/pump schedules depending on relative humidity and temperature at sensor node.
+      A1 , \field Name
+          \required-field
+          \type alpha
+          \reference SystemAvailabilityManagers
+      A2 , \field Sensor Node Name
+          \note The air node where the temperature and the relative humidity are monitored
+          \required-field
+          \type node
+      N1 , \field Relative Humidity
+          \note When the sensor node relative humidity is lower than this RH and the sensor node temperature is higher than the temperature specified in N2, then the system is turned on.
+          \required-field
+          \type real
+          \units dimensionless
+      N2 ; \field Temperature
+          \note When the sensor node temperature is higher than this temperature and the sensor node RH is lower than the RH specified in N1, then the system is turned on.
+          \required-field
+          \type real
+          \units C
```

Comment:
Since one of the temperatures used to control this type of equipment would be zone temperature, which changes seasonally, this temperature field should probably be a schedule. Using that same reasoning, all these air condition inputs could be schedules for added flexibility.

Reply:
make sense. We can discuss this in the technicality meeting.